Cargo.lock

```diff
@@ -438,6 +482,21 @@ dependencies = [
  "thiserror",
 ]
 
+[[package]]
+name = "ring"
```

Comment:
looks like ring's MSRV is still 1.63 but in the past they've been pretty reckless with bumping it. What is it needed for? I thought bdk_electrum used native TLS.

Reply:
I think this is needed by the electrum-client v0.19 dependency of bdk_electrum.

In electrum-client v0.21, it seems there's a way to avoid it:
https://github.com/bitcoindevkit/rust-electrum-client/blob/0.21.0/Cargo.toml#L45-L46

But there aren't any bdk_electrum releases using this version yet.

Related to this, shall we plan an upgrade of our bitcoin dependency to v0.32 in order to use later versions of the bdk_electrum and electrum-client crates?

Reply:
I've created https://github.com/wizardsardine/liana/issues/1224 for upgrading our bitcoin dependency.